#5 [Adding song reproduction]

Efgar opened the pull request:
Story: https://github.com/TFFLabs/SoundFoundry/issues/5

Commit:
#5 [Adding song reproduction]

Story: https://github.com/TFFLabs/SoundFoundry/issues/5

Shinkei commented on src/app/interceptors/errorInterceptor.ts:
```diff
@@ -31,7 +31,7 @@ export class ErrorInterceptor implements HttpInterceptor {
             duration: 1500
           });
-          snackBarRef.afterDismissed().subscribe(() => {
+          snackBarRef.afterDismissed().subscribe(value => {
```
parece que no se esta usando este value

Shinkei commented on src/app/services/playlist.service.ts:
```diff
@@ -1,283 +1,54 @@
-    return json.tracks.map(function(value) {
-      return new Track().deserialize(value);
-    });
+    return [];
```
porque retorna un array vacio?

Efgar replied:
Placeholder, la idea es que aca se llene lo que llega de firebase cuando funcione, quizas desaparezca la funcion a corto plazo

Shinkei replied:
ok ya entiendo well done :ok_hand:

Commit:
#5 [Fixing small issue]

Story: https://github.com/TFFLabs/SoundFoundry/issues/5

some similar fixes will be included in a future PR
